release-guide.md

```diff
@@ -0,0 +1,49 @@
+# Release Guide
```

Comment:
I think we should put this in our wiki instead of in Github.  This isn't relevant to non-Mesosphere contributers